scalafix-core/shared/src/main/scala/scalafix/internal/config/DisableUnlessConfig.scala

```diff
@@ -0,0 +1,76 @@
+package scalafix.internal.config
+
+import metaconfig.ConfDecoder
+import metaconfig.Configured.{NotOk, Ok}
+import org.langmeta.Symbol
+
+import scalafix.internal.config.MetaconfigPendingUpstream.XtensionConfScalafix
+import scalafix.internal.util._
+
+sealed trait UnlessMode
+case object UnlessInside extends UnlessMode
+case object UnlessOutside extends UnlessMode
+
+case class UnlessConfig(
+    mode: UnlessMode,
+    block: Symbol.Global,
+    symbol: Symbol.Global,
+    message: Option[String])
+
+object UnlessConfig {
+  implicit val decoder: ConfDecoder[UnlessConfig] =
+    ConfDecoder.instanceF[UnlessConfig] { c =>
+      (
+        (c.get[String]("mode") match {
```

Comment:
The convention would be to put an `ConfDecoder[UnlessMode]` in the companion object of `UnlessMode` and then `c.get[UnlessMode]("mode")`

Reply:
I've just removed that setting